assets/src/edit-story/components/canvas/multiSelectionMovable.js

```diff
@@ -170,7 +171,19 @@ function MultiSelectionMovable({ selectedElements }) {
           );
         }
       }
-      updateElementsById({ elementIds: [element.id], properties });
+      updates[element.id] = properties;
+    });
+    updateElementsById({
+      elementIds: Object.keys(updates),
+      properties: (currentProperties) => {
+        if (!updates[currentProperties.id]) {
+          return currentProperties;
+        }
+        return {
+          ...currentProperties,
```

Comment:
I think the `updateElementsById()` does this `{...current, ...new}` for you anyway. The `currentProperties` here are only available to (a) give you an ID, and (b) to allow you to calculate a new value based on the previous. E.g. to apply an aspect ratio. But I think here your updates are just an override, so you can just do `return updates[currentProperties.id]`